packages/component-base/src/resize-mixin.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { dedupingMixin } from '@polymer/polymer/lib/utils/mixin.js';
+
+const observer = new ResizeObserver((entries) => {
+  setTimeout(() => {
```

Comment:
Is there a specific reason for using a timeout instead of requestAnimationFrame?

Reply:
Looks like some components were using `setTimeout` previously:

https://github.com/vaadin/web-components/blob/32fbd4a36ed11c7aaceab03a678a43422d342380/packages/avatar-group/src/vaadin-avatar-group.js#L364-L367

IIRC for avatar group this could be caused by `dom-repeat` usage but I'm not 100% sure. 
Let's keep the current approach for now, and reconsider later if needed.